tests/DotNetLightning.Core.Tests/TransactionTests.fs

```diff
@@ -1,18 +1,146 @@
+            MaxHTLCValueInFlightMSat = 10_000_000L |> LNMoney
+            ChannelReserveSatoshis = 1000L |> Money.Satoshis
+            HTLCMinimumMSat = 1000L |> LNMoney
+            ToSelfDelay = 144us |> BlockHeightOffset16
+            MaxAcceptedHTLCs = 1000us
+            ChannelPubKeys = remoteChannelPubKeys
+            Features = FeatureBits.Zero
+            MinimumDepth = 6u |> BlockHeightOffset32
+        }
+        let feeRate = FeeRatePerKw (rand.Next(0, 300) |> uint32)
+        let localAmount = 2_000_000_000L |> LNMoney
+        let remoteAmount = LNMoney.Satoshis(fundingAmount.Satoshi) - localAmount
+        let commitmentSpec = {
+            HTLCs = Map.empty
+            FeeRatePerKw = feeRate
+            ToLocal = localAmount
+            ToRemote = remoteAmount
+        }
+
+        let unsignedCommitmentTx =
+            makeCommitTx
+                fundingScriptCoin
+                commitmentNumber
+                localChannelPubKeys.PaymentBasepoint
+                remoteChannelPubKeys.PaymentBasepoint
+                true
+                localParams.DustLimitSatoshis
+                remoteCommitmentPubKeys.RevocationPubKey
+                localParams.ToSelfDelay
+                localCommitmentPubKeys.DelayedPaymentPubKey
+                remoteCommitmentPubKeys.PaymentPubKey
+                localCommitmentPubKeys.HtlcPubKey
+                remoteCommitmentPubKeys.HtlcPubKey
+                commitmentSpec
+                Network.RegTest
+        let commitmentTx =
+            unsignedCommitmentTx.Value
+                .SignWithKeys(localChannelPrivKeys.FundingPrivKey.RawKey(), remoteChannelPrivKeys.FundingPrivKey.RawKey())
+                .Finalize()
+                .ExtractTransaction()
+
+        let transactionBuilderOpt =
+            ForceCloseFundsRecovery.tryGetFundsFromLocalCommitmentTx
+                localParams
+                remoteParams
+                fundingScriptCoin
+                localChannelPrivKeys
+                Network.RegTest
+                commitmentTx
+        let transactionBuilder = transactionBuilderOpt.Value
+
+        let recoveryTransaction =
+            transactionBuilder
+                .SendAll(localDestPubKey)
+                .BuildTransaction(true)
+        let inputs = recoveryTransaction.Inputs
+        Expect.equal inputs.Count 1 "wrong number of inputs"
+        let input = inputs.[0]
+        Expect.equal input.Sequence.Value (uint32 localParams.ToSelfDelay.Value) "wrong sequence nuber"
+        Expect.equal input.PrevOut.Hash (commitmentTx.GetHash()) "wrong prevout hash"
+        let expectedAmount =
+            let fullAmount = commitmentSpec.ToLocal.ToMoney()
+            let fee = commitmentTx.GetFee [| fundingScriptCoin |]
+            fullAmount - fee
+        let actualAmount =
+            commitmentTx.Outputs.[input.PrevOut.N].Value
+        Expect.equal actualAmount expectedAmount "wrong prevout amount"
+
+    (*
```

Comment:
We should probably just remove this test.
It was originally ported from https://github.com/ACINQ/eclair/blob/f202587e9c71b2106194c0e6cabc8032f6f67df4/eclair-core/src/test/scala/fr/acinq/eclair/transactions/TransactionsSpec.scala#L111

But I guess it is better to add it again when we encounter the need.